### PR TITLE
introduce fixtures and helper functions

### LINKED ISSUE #142: more conu awesomeness (pytest fixtures, generic test cases)
We have some work to do to enable contributors send us code. Just talked to @hhorak and he would like to send us:

 * pytest fixtures
 * generic test cases

Hence work to be done by us:

* [x] add a way to add pytest fixtures in conu
  * [x] should be a "plugin" - explicit installation via RPM/~~PyPI~~
  * [x] please do NOT name it `conu.pytest` - let's not clobber pythonpath
  * [x] add there some very simple example - backend!
* [x] add a way to add generic functions (=test cases) in conu
  * probably just create a namespace `conu.validate`?
  * again, some simple example would be so nice (look at what we have so far)
* [x] ping Honza once ~~merged~~ proposed

### PR DESCRIPTION
**Pull request by TomasTomecek**
TODO:
 * [x] documentation
 * [ ] is `conu.helpers` okay?
 * [ ] how will users import these functions?
 * [x] python packaging
 * [x] rpm packaging

Fixes #142

### CODE REVIEW

File: tests/integration/test_backend.py [hunk 1]
```diff
@@ -68,3 +69,7 @@ def scope():
         assert b_tmp == t
 
     assert not os.path.isdir(t)
+
+
+def test_backend_fixture(docker_backend):
+    assert isinstance(docker_backend, DockerBackend)
```
Comment (dhodovsk) [inline tests/integration/test_backend.py]: nit: '\n'

File: conu/helpers/docker_backend.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from conu import DockerRunBuilder
+
+
+def get_container_output(backend, image_name, command, image_tag="latest",
```
Comment (dhodovsk) [inline conu/helpers/docker_backend.py]: This (and similar) is what will make conu even more awesome :)

**Comment by TomasTomecek** [comment]
@dhodovsk I agree, that should be done in https://github.com/fedora-modularity/conu/issues/144 once `s/_clean/clean/` is done.

**Comment by TomasTomecek** [comment]
Tests are now passing for me locally.

**Comment by dhodovsk** [comment]
For the remaining two points, `from conu.helpers import get_container_output` sounds reasonable. Can this be merged?

**Comment by TomasTomecek** [comment]
> Can this be merged?

Yup.

We might revise the imports in future.

**Pull request merged**